.gitignore

```diff
@@ -1,5 +1,2 @@
-
 app/src/deadlink.config.local.inc.php
-/mariadb/
-/vendor/
```

Comment:
Why were these two lines removed?  This will cause people like me to accidentally commit the composer dependencies when they shouldn't be committed and other participating devs to inadvertently commit their local DB to the repo.

Reply:
Fixed `vendor`. Regarding `mariadb`, it is no longer used because we're now using a Docker volume to store database files.